### PR TITLE
Adding support for Hungarian localisation

### PR DESCRIPTION
**Pull request by robertfodor**

### CODE REVIEW

**Review by dan-weiss — CHANGES_REQUESTED** [review]
Please update the version on line 7 and 11 to 2.16 and update the date on lines 7 and 10 to today's date

File: apa7/apa7.dtx [hunk 1]
```diff
@@ -3872,6 +3874,23 @@ The detailed results are shown in Table~\ref{tab:DeckedTable}. \lipsum[22]
 %\end{macro}
 %
 %
+%\begin{macro}{APA7hungarian.txt}
+%    \begin{macrocode}
+%<*hungarian>
+\ProvidesFile{APA7hungarian.txt}[2022/07/25 APA7 configuration for Hungarian - added by Robert Fodor]
```
Comment (dan-weiss) [inline apa7/apa7.dtx:3880]:
```suggestion
\ProvidesFile{APA7hungarian.txt}[2022/07/25 v1.00 apa7 configuration for Hungarian - added by Robert Fodor]
```

**Comment by dan-weiss** [comment]
@robertfodor Thanks for adding this to the package! It is great to have community support.

**Comment by robertfodor** [comment]
Changed date and version in line 7 and 10.

**Review by dan-weiss — APPROVED** [review]

**Pull request merged**